plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/Utils.php

```diff
@@ -84,13 +84,20 @@ public static function get_quantity_input_args( $product ) {
 	}
 
 	/**
-	 * Make the quantity input interactive by wrapping it with the necessary data attribute.
+	 * Make the quantity input interactive by wrapping it with the necessary data attribute and adding an input event listener.
 	 *
 	 * @param string $quantity_html The quantity HTML.
 	 * @param string $wrapper_attributes Optional wrapper attributes.
 	 * @return string The quantity HTML with interactive wrapper.
 	 */
 	public static function make_quantity_input_interactive( $quantity_html, $wrapper_attributes = '' ) {
+		$html = new \WP_HTML_Tag_Processor( $quantity_html );
```

Comment:
`$html` here and `$quantity_html` hold different type of value but has the similar name, should we use other name for the Tag Processor instance?

Reply:
Updated it (alongside some other instances) in a29bbeb5b2ea39afd953002a71dbc56e690a35d5.